contracts/SpokePool.sol

```diff
@@ -242,7 +242,9 @@ abstract contract SpokePool is Testable, Lockable, MultiCaller {
         emit FilledRelay(relayHash, relayFills[relayHash], repaymentChain, amountToSend, msg.sender, relayData);
     }
 
-    function initializeRelayerRefund(bytes32 relayerRepaymentDistributionProof) public {}
```

Comment:
nit to give me the required interface. @nicholaspai 's [PR](https://github.com/across-protocol/across-smart-contracts-v2/pull/20) implements this actual logic.